backend/src/hatchling/builders/wheel.py

```diff
@@ -178,10 +192,11 @@ def default_file_selection_options(self) -> FileSelectionOptions:
         if include or packages or only_include:
             return FileSelectionOptions(include, exclude, packages, only_include)
 
-        for project_name in (
+        project_names = {
```

Comment:
Can we please build up a `list` instead for the average case where the first one will work?